packages/radio-group/src/vaadin-radio-button.js

```diff
@@ -80,13 +80,12 @@ class RadioButton extends SlotLabelMixin(
         }
 
         .vaadin-radio-button-container {
-          display: inline-flex;
+          display: flex;
           align-items: baseline;
-          width: 100%;
+          position: relative;
```

Comment:
I assume this is fixing some other issue, and not directly related to this clickable area?

Reply:
I had to move this CSS here to extend the slotted `input` to cover the full height in multi-line radio-button.
We probably can revert this change, together with setting hardcoded width on the input.

Reply:
Updated to use CSS grid instead. Looks like that allows to extend `input` height.

Reply:
> We probably can revert this change, together with setting hardcoded width on the input.

Yeah, sounds good to me.

Reply:
@jouni How about changing to `display: grid`? This makes it possible to extend `input` to full height.